src/types/express/AuthenticatedRequest.ts

```diff
@@ -20,6 +21,12 @@ interface ObjectWithAuthWithRealmAccessRoles {
 	};
 }
 
+interface ObjectWithAuthWithOrganizations {
+	auth: {
+		organizations: Record<string, { id: KeycloakId }>;
```

Comment:
I was thinking this should be a list, but you're right, Keycloak does it as an object with organization short names as keys in an object.